src/pages/CreateProposal/index.tsx

```diff
@@ -0,0 +1,217 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import styled from 'styled-components'
+import JSBI from 'jsbi'
+import { utils } from 'ethers'
+import { Button, TYPE } from 'theme'
+import { Currency, CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { UNI } from '../../constants/tokens'
+import AppBody from '../AppBody'
+import { generateBytesByType } from 'utils/generateBytesByType'
+import { CreateProposalTabs } from '../../components/NavigationTabs'
+import { ButtonError } from 'components/Button'
+import { AutoColumn } from 'components/Column'
+import { BlueCard } from 'components/Card'
+import { Wrapper } from 'pages/Pool/styleds'
+import { ProposalAction, ProposalActionSelector, ProposalActionSelectorModal } from './ProposalActionSelector'
+import { ProposalEditor } from './ProposalEditor'
+import { ProposalActionDetail } from './ProposalActionDetail'
+import { ProposalSubmissionModal } from './ProposalSubmissionModal'
+import { useActiveWeb3React } from 'hooks/web3'
+import {
+  CreateProposalData,
+  ProposalState,
+  useCreateProposalCallback,
+  useLatestProposalId,
+  useProposalData,
+  useProposalThreshold,
+  useUserVotes,
+} from 'state/governance/hooks'
+import { Trans } from '@lingui/macro'
+
+const ProposalWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  margin-top: 10px;
+`
+
+const CreateProposalButton = ({
+  hasActiveOrPendingProposal,
+  hasEnoughVote,
+  isFormInvalid,
+  handleCreateProposal,
+}: {
+  hasActiveOrPendingProposal: boolean
+  hasEnoughVote: boolean
+  isFormInvalid: boolean
+  handleCreateProposal: () => void
+}) => {
+  return hasActiveOrPendingProposal ? (
+    <ButtonError marginTop="18px">
```

Comment:
does `ButtonError` also disables the button?

Reply:
Nope, I just realized that i need to add error={true} prop to actually have it show the error style, so currently it's just a no op button. I'm checking with Cal because I'm not sure when to use button disable vs. ButtonError.